src/cdk-experimental/menu/menu-bar.ts

```diff
@@ -248,6 +256,11 @@ export class CdkMenuBar extends CdkMenuGroup implements Menu, AfterContentInit,
     return this.orientation === 'horizontal';
   }
 
+  // In Ivy the `host` metadata will be merged, whereas in ViewEngine it is overridden. In order
+  // to avoid double event listeners, we need to use `HostListener`. Once Ivy is the default, we
```

Comment:
You'll need a stronger condition for shifting it back -- I'd say ViewEngine needs to be *deprecated* before you could do this.

Reply:
This is the comment we have throughout the library for this pattern, we can leave it as is (even though it's actually that ViewEngine needs to be _deleted_ before we can do this)